tests/integration/rag/test_query_generator.py

```diff
@@ -76,32 +76,33 @@ def query_generator(app_models):
 def test_generate_queries(
     input_query, expected_queries, query_generator, correctness_metric
 ):
-    # When
+    # When: the query generator's generate_queries method is invoked
     result = query_generator.generate_queries(input_query)
 
-    # Then
+    # Then: we evaluate the test case using deepeval metrics
     test_case = LLMTestCase(
         input=input_query,
         actual_output=str(result.queries),
         expected_output=str(expected_queries["queries"]),
     )
-
-    # Assert using DeepEval
     eval_results = evaluate(
```

Comment:
Add a comment to explain what "correctness" checks?

Reply:
I think correctness metric is self-explanatory itself.